db/migrate/20170504145330_sync_whitehall_public_updated_at.rb

```diff
@@ -0,0 +1,29 @@
+class SyncWhitehallPublicUpdatedAt < ActiveRecord::Migration[5.0]
+  def change
+    schema_names = %w(gone redirect special_route speech statistical_data_set take_part topical_event_about_page)
+
+    schema_names.each do |schema_name|
+      scope = Edition.where(publishing_app: "whitehall")
+                     .where(schema_name: schema_name)
+                     .where("first_published_at AT TIME ZONE 'Europe/London' > public_updated_at")
```

Comment:
Do we need the `AT TIME ZONE` bit for `public_updated_at` as well, or is that implied, or simply not required?